TwitchPlaysAssembly/Src/UI/ModuleCameras.cs

```diff
@@ -428,15 +444,19 @@ public void SetNotes()
 					(numNameless > 0 && numNamed > 0 ? " + " : "") +
 					TwitchGame.Instance.CommandQueue.Where(c => c.Name != null).Select(c => $"“{c.Name}”").Join(", ");
 				NotesTextBackgrounds[ix].color = Color.HSVToRGB(hue, .246f, .93f);
```

Comment:
Looking at this, this line should probably be deleted. It looks like the color is set directly below it.

Reply:
whoops, mistake solving the conflict, thanks for noticing

Reply:
whoops, made a mistake while solving the conflict, thanks for noticing